source/_docs/z-wave/adding.markdown

```diff
@@ -90,13 +67,28 @@ If your device isn't responding to this process, possibly because you've factory
     ```
 
 5. Click **Set State**
-6. Go to the Z-Wave control panel in the Home Assistant frontend
-7. Click the **Remove Failed Node** button in the *Z-Wave Node Management* card
+6. Go to the [Z-Wave control panel](/docs/z-wave/control-panel/)
+7. Click **Remove Failed Node** in the *Z-Wave Node Management* card
 8. The device will now be removed, but that won't show until you restart Home Assistant
 
-## Troubleshooting
+## Network Key
```

Comment:
The network key section is already described on the installation page. Let's remove it from here (as it is a duplicate).

Reply:
done.